safe_transaction_service/history/services/index_service.py

```diff
@@ -353,7 +353,38 @@ def _reprocess(self, addresses: List[str]):
             queryset = queryset.filter(internal_tx___from__in=addresses)
         queryset.update(processed=False)
 
-    def reprocess_addresses(self, addresses: List[str]):
+    @transaction.atomic
+    def fix_out_of_order(self, address: ChecksumAddress, internal_tx: InternalTx):
+        """
+        Fix a Safe that has transactions out of order (not processed transactions in between processed ones, usually
+        due a reindex)
+
+        :param address:
+        :return:
```

Comment:
nitpick: we are missing the `internal_tx` here

Reply:
You are right, done!